### PR TITLE
docs(README): add `meteor` to `huh?` section

### PR DESCRIPTION
**Pull request by stefanlogue**
Adding my project `meteor` to the README. 

The tool is built using `huh?`

### CODE REVIEW

**Comment by wissam** [comment]
Thank you @stefanlogue , will add it soon. We just need to iron out a bit the gitlab actions in here.

**Review by Delta456 — COMMENTED** [review]
We would need to follow the lint rules for now.

File: README.md [hunk 1]
```diff
@@ -17,13 +17,18 @@ To contribute, see [CONTRIBUTING.md](./CONTRIBUTING.md)
 
 - [Applications](#applications)
   - [Bubbletea](#bubbletea)
+  - [huh?](#huh?)
```
Comment (Delta456) [inline README.md:20]:
```suggestion
  - [Huh?](#huh)
```

File: README.md [hunk 2]
```diff
@@ -17,13 +17,18 @@ To contribute, see [CONTRIBUTING.md](./CONTRIBUTING.md)
...
   - [Editor Plugins](#editor-plugins)
 
 ## Applications
 
 ### Bubbletea
 
+### `huh?`
+
+- [meteor](https://www.github.com/stefanlogue/meteor): A highly configurable CLI tool for writing conventional commits, built with `huh?`!
+
```
Comment (Delta456) [inline README.md:31]:
```suggestion
### huh

- [meteor](https://www.github.com/stefanlogue/meteor) - A highly configurable CLI tool for writing conventional commits, built with `huh?`!

```

**Comment by wissam** [comment]
Closing the pull request, meteor is already in :)

**Comment by stefanlogue** [comment]
Should this have been closed? Meteor is made with huh, not bubble tea

**Comment by Delta456** [comment]
> Should this have been closed? Meteor is made with huh, not bubble tea

Meteor indirectly uses Bubble Tea https://github.com/stefanlogue/meteor/blob/main/go.mod#L19

**Pull request closed without merging**